applications/minotari_console_wallet/src/ui/state/app_state.rs

```diff
@@ -730,11 +730,11 @@ impl AppStateInner {
                 .transaction_service
                 .get_pending_outbound_transactions()
                 .await?
-                .values()
+                .iter()
                 .map(|t| CompletedTransaction::from(t.clone()))
                 .collect::<Vec<CompletedTransaction>>(),
         );
-
+        // Question(C): We sort txs by mined_timestamp in sql, here is by timestamp
```

Comment:
```suggestion
```
This is probably much of a muchness. Timestamp = creation date. 
Creation date != mined date. 
Only weird caveat is that mined timestamp could be null, which sqlite will then put at the bottom of the list. If you just created a tx you would want to see it at the top. 

On the view only wallet, in universe, these two values will probably be the same, as the wallet does not talk to other wallets to create transactions